### DISCUSSION TITLE
ScriptReference/AssetDatabase.Refresh

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#256] [Notes]
# ScriptReference/AssetDatabase.Refresh

https://docs.unity3d.com/ScriptReference/AssetDatabase.Refresh.html

**Comment by radiatoryang** [comment]
if you just want to refresh one (or a few) assets, you can use [`AssetDatabase.ImportAsset()`](https://docs.unity3d.com/ScriptReference/AssetDatabase.ImportAsset.html) instead
